libraries/ESP8266WiFi/src/WiFiServer.h

```diff
@@ -100,6 +100,8 @@ class WiFiServer {
   uint16_t port() const;
   void close();
   void stop();
+  void end();
+  virtual operator bool();
```

Comment:
`virtual` / non-virtual here?
I see at least esp32 has a normal one, note of the difference

Reply:
it doesn't have to be virtual. I removed `virtual`